platform/posix/transport/utest/reconnect_utest.c

```diff
@@ -0,0 +1,128 @@
+#include <string.h>
+#include <stdbool.h>
+#include <stdlib.h>
+#include "/usr/include/errno.h"
```

Comment:
This is part of the ISO C99 standard library at least (We don't have issues with having tests use C99)
```suggestion
#include <errno.h>
```

Reply:
Keep in mind that this will require me to `set( CMAKE_C_FLAGS "-isystem /usr/include ${CMAKE_C_FLAGS}" )` because there are errno.h files that seem to just include itself... The right one is at /usr/include, and I'm not sure why the other ones exist. I've also updated the other ones.